lib/ranked-model/ranker.rb

```diff
@@ -183,7 +183,7 @@ def assure_unique_position
 
       def rearrange_ranks
         _scope = finder
-        escaped_column = ActiveRecord::Base.connection.quote_column_name ranker.column
+        escaped_column = instance_class.ranker_connection_class.connection.quote_column_name ranker.column
```

Comment:
This call becomes quite clunky with the abstraction.

Reply:
it should probably just be instance_class anyhow

Reply:
Agreed. Let's go with that. Especially given a particular model may have a different connection.